source/val/validate_instruction.cpp

```diff
@@ -483,6 +483,22 @@ spv_result_t InstructionPass(ValidationState_t& _, const Instruction* inst) {
     if (auto error = LimitCheckNumVars(_, inst->id(), storage_class)) {
       return error;
     }
+  } else if (opcode == SpvOpSamplerImageAddressingModeNV) {
+    if (!_.HasCapability(SpvCapabilityBindlessTextureNV)) {
+      return _.diag(SPV_ERROR_MISSING_EXTENSION, inst)
+             << "OpSamplerImageAddressingModeNV supported only with extension "
+                "SPV_NV_bindless_texture";
+    }
+    uint32_t bitwidth = inst->GetOperandAs<uint32_t>(0);
+    if (_.samplerimage_variable_address_mode() != 0) {
```

Comment:
@alan-baker this is where it checks to ensure OpSamplerImageAddressingModeNV is called only once. Is this enough?

Reply:
I think so, can you just cover it with a test?

Reply:
added a test "NVBindlessAddressModeFromLayoutSpecifiedTwice" in val_layout_test.cpp